indico/modules/events/registration/plugins.py

```diff
@@ -0,0 +1,21 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2022 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+class CaptchaPluginMixin:
```

Comment:
I would not put this (or any other generic captcha code not specific to registrations) in the registration module. Captchas may be useful for Indico account creation as well.

Reply:
Yeah I was debating where best to put this..